src/Service/SF1500/SF1500.php

```diff
@@ -420,58 +396,135 @@ public function getEnhedNavnOgOverordnetOrganisationsId($organisationEnhedsId, $
     }
 
     /**
-     * Fetches bruger and organisation funktions id for nearest manager or null if no manager exists.
+     * Fetches bruger and organisation funktions id for managers from user id.
+     * Returns null if no manager exists.
      */
-    public function getNearestManagerBrugerAndFunktionsId($userId, $managerFunktionsTypeId)
+    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId)
     {
         $token = $this->fetchSAMLToken();
```

Comment:
It seems cumbersome to pass the token around to functions in this class (I assume it's always the same token).

Will it make sense to create a convenience function like

```php
private function buildHeaderXML(string $endpoint, string $action) {
  $token = $this->fetchSAMLToken();
  return $this->xmlBuilder->buildHeaderXML($endpoint, $action, $token);
}
```

say, to make this easier to handle?

Reply:
This makes great sense. Fixed.